docs/content/concepts/graphql-rpc.mdx

```diff
@@ -15,6 +15,14 @@ Jump to the following sections for more details:
 
 For more details on GraphQL fundamentals, see the introductory documentation from [GraphQL](https://graphql.org/learn/) and [GitHub](https://docs.github.com/en/graphql/guides/introduction-to-graphql).
 
+:::info
+
+Refer to [Access Sui Data](/guides/developer/getting-started/data-serving.mdx) for an overview of options to access Sui network data.
+
+GraphQL RPC was earlier shared to be in beta in 2024. But based on valuable feedback from the community, its release stage has been updated to alpha. You can refer to the updated high-level timeline for beta and GA releases in the above linked document.
```

Comment:
```suggestion
Based on valuable feedback from the community, the GraphQL RPC release stage has been updated from beta to alpha. Refer to the high-level timeline for beta and GA releases in the previously linked document.
```
Typically want to avoid 'above' and 'below' because you can never be certain of rendered location. Here you don't have to really worry about that but just want to remain consistent in wording.